Gherkin Features/ID026_Accepting_Invitation_To_Events.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Accept invitation to an event
+
+As a OnCampus User
+I would like to delete my existing unique / recurring event
```

Comment:
uhm this seems to be copy pasted from the delete event gherkin... maybe fix that lol